docs/InsertingRazorIntoVSCodeCSharp.md

```diff
@@ -6,98 +6,46 @@ The C# extension is the shipping vehicle for the VS Code Razor experience, so to
 
 At a high level inserting involves:
 
-1. Downloading the right build artifacts from our official CI build
-2. Uploading a subset of those artifacts to a special folder on a network share
-3. Running a pipeline to take the folder contents and upload them to a CDN for external consumption
-4. Doing a pull request to the extension repo to update various download URLs
+1. Find the version number from CI builds
+2. Updating the vscode repo and doing a PR by
+    1. Update the version in the `package.json`
+    2. Running `updateRazorVersion` via gulp
+    3. Pushing a PR
 
 ## Permissions
 
-Before getting started you it should be noted that only Microsoft employees will have the correct access to get the right information to perform and insertion. Details on the permissions required are outlined at this [internal document](https://github.com/dotnet/aspnetcore-internal/blob/main/docs/release/InsertingRazorIntoOmniSharp.md#get-the-permissions-youll-need-later).
+Before getting started you it should be noted that only Microsoft employees will have the correct access to get the right information to perform and insertion. Details on the permissions required are [somewhere](????).
```

Comment:
TODO before merge: find if documentation exists for getting permissions, or if we can have a contact listed.

Reply:
What do we need permissions for? There is nothing in this doc that implies on MS people can do it anyway.

Reply:
The nuget repo that is used is downstream from the repo the package is published to. Without permissions the feed can't pull in a new package

Reply:
There's a note in the detailed steps that indicate where failure will happen without permissions

Reply:
@dibarbet is there guidance I can give people for what permissions they would need for pulling from the upstream?

Reply:
@ryzngard see https://github.com/dotnet/vscode-csharp/blob/main/CONTRIBUTING.md#updating-the-roslyn-language-server-version - `CSharp VS Code Extension contributors`